keps/sig-storage/20190122-csi-inline-volumes.md

```diff
@@ -35,7 +34,8 @@ status: implementable
     * [Persistent vs ephemeral lifecycle setting](#persistent-vs-ephemeral-lifecycle-setting)
     * [VolumeHandle generation](#volumehandle-generation)
     * [Inline CSI volume operation stages](#inline-csi-volume-operation-stages)
-    * [Security considerations](#security-considerations)
+    * [Volume sharing considerations](#volume-sharing-considerations)
```

Comment:
the anchor doesn't link to the actual title ("Persistent inline volume referred by multiple pods")

Reply:
Ha. Missed it. Thanks.